### PR TITLE
Automated cherry pick of #75176: fix kubectl describe hpa does not work at version 1.14

### PR DESCRIPTION
**Pull request by ahmadnurus**
Cherry pick of #75176 on release-1.14.

#75176: make describers of different versions work properly when

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

### CODE REVIEW

**Comment by prksu** [comment]
fix kubernetes/kubectl#750
friendly ping /cc @seans3

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
You still need release manager approval for 1.14 cherry-pick

**Comment by prksu** [comment]
@soltysh thanks, i'll be asking release 1.14 team for approval

**Pull request merged**
